language-examples/python/vectara-grpc/grpc_basic_operations.py

```diff
@@ -90,15 +90,20 @@ def index(customer_id: int, corpus_id: int, idx_address: str, jwt_token: str):
         try:
             index_stub = services_pb2_grpc.IndexServiceStub(
                 grpc.secure_channel(idx_address, grpc.ssl_channel_credentials()))
-            
+
             # Vectara API expects customer_id as a 64-bit binary encoded value in the metadata of
             # all grpcs calls. Following line generates the encoded value from customer ID.
             packed_customer_id = struct.pack(">q", int(customer_id))
             response = index_stub.Index(index_req,
                                         credentials=grpc.access_token_call_credentials(jwt_token),
                                         metadata=[("customer-id-bin", packed_customer_id)])
-            logging.info("Indexed document successful: %s", response)
+            if response.status.code != status_pb2.StatusCode.OK:
```

Comment:
Let's whitelist ALREADY_EXISTS code? I.e., if the code is ALREADY_EXISTS, then all is OK.

Reply:
Done.